composer.json

```diff
@@ -1,66 +1,39 @@
 {
-    "name": "aakb/itstyr",
-    "description": "",
```

Comment:
Why do you remove this?

Reply:
Changed back. I didnt, just installed a fresh project.